### PR TITLE
[Feature] Support MMCls backbones for TSN 

### PR DESCRIPTION
**Pull request by kennymckormick**
TL;DR: Now you can use backbones in MMCls for TSN with the name "mmcls:backbone_name"

- [x] Model building succeed
- [x] Provide an example

PS: support every backbone in MMCls (except VGG, which will be refactored soon).

### CODE REVIEW

**Comment by innerlee** [comment]
Can this be tested?

**Comment by kennymckormick** [comment]
> Can this be tested?

Yes, I have manually tested every backbone in MMCls (there's not much). Besides, I will add the corresponding unit test.

**Comment by kennymckormick** [comment]
@innerlee

File: configs/recognition/tsn/README.md [hunk 1]
```diff
@@ -54,6 +54,18 @@
 
 Here, We use [1: 1] to indicate that we combine rgb and flow score with coefficients 1: 1 to get the two-stream prediction (without applying softmax).
 
+### Using backbones from 3rd-party in TSN
+
+It's possible and convenient to use a 3rd-party backbone for TSN under the framework of MMAction2, here we provide some examples for:
+
+- [x] Backbones from MMClassification
+- [ ] Backbones from torchvision
```
Comment (innerlee) [inline configs/recognition/tsn/README.md:62]: Remove torchvision and others for now?

Comment (kennymckormick) [reply configs/recognition/tsn/README.md]: OK

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
We will support importing any object from any mm-series repos in future. Before that, I think the current pr is an okay solution.

**Comment by kennymckormick** [comment]
> We will support importing any object from any mm-series repos in future. Before that, I think the current pr is an okay solution.

What will the code look like? Importing from MMCV?

**Pull request merged**